sdk/eventhub/event-hubs/rollup.base.config.js

```diff
@@ -127,7 +120,6 @@ export function browserConfig(test = false) {
 
       cjs({
         namedExports: {
-          events: ["EventEmitter"],
```

Comment:
Any idea why we needed this before? I know rhea-promise uses EventEmitter heavily

Reply:
I'm not sure why it was needed before, but the tests using the rollup bundle still pass and I also ran the browser sample using webpack and that worked too.

I did do some work in the past to make `rhea` more bundler friendly so perhaps that made this no longer necessary.